### ISSUE TITLE
testkube regression: helm chart fails to install with flux due to duplicate yaml keys

### ISSUE THREAD

**Issue by gberche-orange** [kubeshop/helm-charts#453]
**Describe the bug**

#449 introduced a regression with duplicate yaml key between the template and default values

https://github.com/kubeshop/helm-charts/blob/68dd5e2801dc26a62a3cff011e13ec299e645195/charts/testkube-api/templates/deployment.yaml#L178-L183

https://github.com/kubeshop/helm-charts/blob/68dd5e2801dc26a62a3cff011e13ec299e645195/charts/testkube-api/values.yaml#L314-L318

As a result, installation from fluxcd fails with message

```
      Helm upgrade failed: error while running post render on files: map[string[]interface {}(nil): yaml: unmarshal errors:
        line 107: mapping key "httpGet" already defined at line 100
        line 118: mapping key "httpGet" already defined at line 112
```

See related https://github.com/fluxcd/helm-controller/issues/283#issuecomment-1277624223

While duplicate keys are mostly ignored by some tools but not others, this problem is unfortunately not detected by `helm lint` (once #452 gets included)

```
$ helm lint
==> Linting .
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```

More background explaining why fluxcd fails with duplicated yaml keys can be found at https://github.com/kubernetes-sigs/kustomize/issues/3480

I searched a while how to automatically detect such regressions in the future in the `kubeshop/helm-charts` github workflows:
* some python libraries can detect duplicated yaml keys
* the `sigs.k8s.io/kustomize/kyaml` cli is likely to detect such errors

https://github.com/kubernetes-sigs/kustomize/blob/2fa944b1cdc1f9ad4f72b3df8e67553e0b97d37d/cmd/kyaml/README.md
> This package exists to expose kyaml filters directly as cli commands for the purposes of development of the kyaml package and as a reference implementation for using the libraries.

I for now failed to install kyaml (likely by lack of skills related go build environment)

```
go install -v sigs.k8s.io/kustomize/cmd/kyaml@latest
go: sigs.k8s.io/kustomize/cmd/kyaml@latest: module sigs.k8s.io/kustomize@latest found (v2.0.3+incompatible), but does not contain package sigs.k8s.io/kustomize/cmd/kyaml
```

**To Reproduce**

```
helm install --dry-run testkube-manual kubeshop/testkube --version 1.10.272  | tail -n +7 ../testkube.yaml | head -n -3 > testkube.yaml
```

Find duplicate keys

```yaml
          readinessProbe:
            failureThreshold: 3
            httpGet:
              path: /health
            initialDelaySeconds: 30
            periodSeconds: 15
            successThreshold: 1
            timeoutSeconds: 10
            httpGet:
              path: /health
              port: 8088
```

** workaround **

Revert to 1.10.256

**Expected behavior**

**Version / Cluster**
- Which testkube version? helm chart 1.10.257 to 1.10.272
- What Kubernetes cluster? (e.g. GKE, EKS, Openshift etc, local KinD, local Minikube)
- What Kubernetes version?

**Screenshots**
If applicable, add CLI commands/output to help explain your problem.
 
**Additional context**

**Comment by ypoplavs** [comment]
This issue was fixed in our new release version. Please upgrade to the latest version and let us know in case of anything.

**Comment by gberche-orange** [comment]
thanks @ypoplavs for the fix in a21af64 ! This indeed solves the reported issue.
